nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -123,6 +126,10 @@ function closure() {
 }
 
 system_closure="$tmpdir/system.closure"
+# Create a FIFO — this prevents this from breaking when $tmpdir is on a small
+# tmpfs (e.g. in image-building VMs) by essentially piping straight into
+# nix-store --import as called in nixos-prepare-root
```

Comment:
`this prevents this` sounds strange. 

From the perspective of a future reader the following might be easier to read:

```
# Pipes `nix-store --export` into `nix-store --import` using a FIFO to save disk io/temporary disk space.
# `nix-store --import` will be called from nixos-prepare-root.
```